lib/Data/Database.php

```diff
@@ -233,37 +252,41 @@ public function read($pasteid)
         }
 
         self::$_cache[$pasteid] = false;
+        $rawData                = '';
         try {
             $paste = self::_select(
                 'SELECT * FROM ' . self::_sanitizeIdentifier('paste') .
                 ' WHERE dataid = ?', array($pasteid), true
             );
+            if ($paste !== false) {
+                $rawData = self::$_type === 'oci' ? self::_clob($paste['DATA']) : $paste['data'];
```

Comment:
Hmm you use clob here. I don't really know the details of what that is or how it is saved in Oracle DBs, but [this SO question](https://stackoverflow.com/questions/36200534/pdo-oci-into-a-clob-field) indicates you need to save the data as  the type `PARAM_LOB` then?

In the query above, you did not save the data as such, so why/how do you need this here now?

Reply:
in the `bindParam` query I mean… see  https://www.php.net/manual/de/pdo.constants.php

Reply:
Ohhh now I properly read that SO question and AFAIK you did actually apply that workaround there… Uff that is crazy stuff, but looks fine on the first look.